lib/index.js

```diff
@@ -21,7 +21,8 @@ const defaults = {
 	name: 'casket',
 	root: process.cwd(),
 	readonly: false,
-	nodelete: false,
+	'no-delete': false,
```

Comment:
I'd prefer `noDelete` and `noUpload` here, as this part of the code is not related to the command line interface and should therefore have idiomatic variable names.

Reply:
That's a very fair observation, fixed!